src/test/kotlin/com/nexters/gamss/card/service/CardDeleteConcurrencyIntegrationTest.kt

```
@@ -170,4 +174,67 @@ class CardDeleteConcurrencyIntegrationTest {
+                    { conversationService.deleteConversation(memberId, conversation.id) },
+                    { conversationService.deleteConversations(memberId, listOf(conversation.id)) },
+                )
+
+            val startLine = CyclicBarrier(tasks.size)
+            val futures =
+                tasks.map { task ->
+                    executor.submit {
+                        startLine.await() // 모든 스레드를 동시에 출발시켜 경합을 유도한다
+                        try {
+                            task()
+                        } catch (e: BusinessException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)